app/mailers/user_mailer.rb

```diff
@@ -86,21 +92,28 @@ def batch_subscription_notification(subscription_id, entries)
     if @creations.count > 1
       subject += " and #{@creations.count - 1} more"
     end
-    
-    mail(
-      :to => @subscription.user.email,
-      :subject => "[#{ArchiveConfig.APP_SHORT_NAME}] #{subject}"
-    )
+    I18n.with_locale(Locale.find(@subscription.user.preference.prefered_locale).iso) do
+      mail(
+        :to => @subscription.user.email,
+        :subject => "[#{ArchiveConfig.APP_SHORT_NAME}] #{subject}"
+      )
+    end
+    ensure
+     I18n.locale = I18n.default_locale
   end
 
   # Emails a user to say they have been given more invitations for their friends
   def invite_increase_notification(user_id, total)
     @user = User.find(user_id)
     @total = total
-    mail(
-      :to => @user.email,
-      :subject => "[#{ArchiveConfig.APP_SHORT_NAME}] New Invitations"
-    )
+    I18n.with_locale(Locale.find(@user.preference.prefered_locale).iso) do
+      mail(
+        :to => @user.email,
+        :subject => "#{t 'user_mailer.invite_increase_notification.subject' ,app_name: ArchiveConfig.APP_SHORT_NAME}"
```

Comment:
I don't know which way around these will be merged so that the code they have in common will no longer appear in the diff for this or the one for Issue 4100, so here is the same comment again:

The space always goes after the comma, never before. 
Please move the space between 'user_mailer.invite_increase_notification.subject' and "app_name:" so that this part reads 'user_mailer.invite_increase_notification.subject', app_name